app/views/wishlists/_item.html.erb

```diff
@@ -1,7 +1,18 @@
-<li>
-  <p>
-    <strong><%= item.name %></strong>
-  </p>
+<div class="text-center">
```

Comment:
Totally not needed `div`. You can a rule to center the text inside your custom SCSS.